src/SparkPayloadEthereum.sol

```diff
@@ -5,7 +5,7 @@ import './AaveV3PayloadBase.sol';
 
 import { Ethereum } from 'spark-address-registry/Ethereum.sol';
 
-import { IL2BridgeExecutor } from 'spark-gov-relay/interfaces/IL2BridgeExecutor.sol';
+import { IExecutor } from 'spark-gov-relay/src/interfaces/IExecutor.sol';
```

Comment:
this is breaking the build for me, I believe it relies on an implicit remapping but we already have an explicit one defined. using `spark-gov-relay/interfaces/IExecutor.sol` works though

Reply:
You need to update the submodules.

```
git submodule update --init --recursive
```

Reply:
nevermind, we don't have any explicit remappings, I simply had a stray remappings.txt in my working tree